web/modules/custom/collection/src/Plugin/Field/FieldFormatter/LastUpdateFormatter.php

```diff
@@ -0,0 +1,206 @@
+        '@name' => $value->label(),
+        '@date' => $this->dateFormatter->format(\Drupal::time()->getRequestTime(), $machine_name),
+      ]);
+    }
+    $date_formats['custom'] = $this->t('Custom');
+
+    $form['tooltip_format'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Tooltip format'),
+      '#description' => $this->t('Select the date/time format to be used for the tooltip.'),
+      '#options' => $date_formats,
+      '#default_value' => $this->getSetting('tooltip_format'),
+    ];
+    $form['tooltip_format_custom'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Tooltip custom format'),
+      '#description' => $this->t('See <a href="http://php.net/manual/function.date.php" target="_blank">the documentation for PHP date formats</a>.'),
+      '#default_value' => $this->getSetting('tooltip_format_custom'),
+      '#states' => [
+        'visible' => [
+          [
+            ':input[name="fields[' . $this->fieldDefinition->getName() . '][settings_edit_form][settings][tooltip_format]"]' => [
+              'value' => 'custom',
+            ],
+          ],
+        ],
+      ],
+    ];
+    $form['timeago_settings'] = [
+      '#type' => 'textarea',
+      '#title' => $this->t('A Yaml representation of settings to be passed to the timeago plugin'),
+      '#desciption' => $this->t('Values passed here will be merged into the timeago plugin defaults. See <a href="http://timeago.yarp.com">timeago.yarp.com</a> for the settings structure.'),
+      '#default_value' => Yaml::encode($this->getSetting('timeago_settings')),
+      '#element_validate' => [[static::class, 'decodeTimeagoSettings']],
+    ];
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $date_format = $this->getSetting('date_format');
+    $tooltip_format = $this->getSetting('tooltip_format');
+    $custom_date_format = $tooltip_format_custom = '';
+    $timezone = $this->getSetting('timezone') ?: NULL;
+    $langcode = $tooltip_langcode = NULL;
+
+    // If an RFC2822 date format is requested, then the month and day have to
+    // be in English. @see http://www.faqs.org/rfcs/rfc2822.html
+    if ($date_format === 'custom' && ($custom_date_format = $this->getSetting('custom_date_format')) === 'r') {
+      $langcode = 'en';
+    }
+    if ($tooltip_format === 'custom' && ($tooltip_format_custom = $this->getSetting('tooltip_format_custom')) === 'r') {
+      $tooltip_langcode = 'en';
+    }
+
+    /** @var \Drupal\rdf_entity\RdfInterface $collection */
+    $collection = $items->getEntity();
+
+    // Add all the node group content once.
+    $tags = Cache::buildTags('og-group-content', $collection->getCacheTagsToInvalidate());
+    // Unfortunately, this doesn't include the solutions.
+    /** @var \Drupal\Core\Field\EntityReferenceFieldItemListInterface $solutions */
+    $solutions = $collection->get('field_ar_affiliates');
```

Comment:
The solution tags are not needed here. Only the collection tags and og collection tags are needed.

Reply:
As, we've discussed on chat we will still keep this until we clarify and we establish a rule on how the cache should invalidate.